refinery/tool_manager/models.py

```diff
@@ -280,14 +281,33 @@ def __str__(self):
         )
 
     def get_input_file_uuid_list(self):
-        # Tools can't be created without the `node_uuid_list` existing so no
+        # Tools can't be created without the `file_uuid_list` existing so no
         # KeyError is being caught
-        return self.get_tool_launch_config()["node_uuid_list"]
+        return self.get_tool_launch_config()["file_uuid_list"]
 
     def get_file_relationships(self):
+        return self.get_tool_launch_config()["file_relationships"]
+
+    def get_file_relationships_urls(self):
+        return ast.literal_eval(
+            self.get_tool_launch_config()["file_relationships_urls"]
+        )
+
+    def get_file_relationship_galaxy_info(self):
+        if self.get_tool_type() != ToolDefinition.WORKFLOW:
+            raise NotImplementedError
+
         return ast.literal_eval(
-            self.get_tool_launch_config()["file_relationships"]
+            self.get_tool_launch_config()["file_relationships_galaxy"]
+        )
+
+    def get_node_uuids(self):
+        tool_launch_config = self.get_tool_launch_config()
+        node_uuids = re.findall(
+            r'[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}',
+            tool_launch_config["file_relationships"]
```

Comment:
Is this a blob of JSON or something? I feel uncomfortable just throwing an RE at it if there really is a structure there.